Description: Add missing format strings
Author: Stephen Kitt <skitt@debian.org>

--- a/src/debug/debug.cpp
+++ b/src/debug/debug.cpp
@@ -752,8 +752,8 @@
 	if ((cpu.pmode) && curSelectorName[0]) {
 		char out1[200], out2[200];
 		GetDescriptorInfo(curSelectorName,out1,out2);
-		mvwprintw(dbg.win_reg,2,28,out1);
-		mvwprintw(dbg.win_reg,3,28,out2);
+		mvwprintw(dbg.win_reg,2,28,"%s",out1);
+		mvwprintw(dbg.win_reg,3,28,"%s",out2);
 	}
 
 	wattrset(dbg.win_reg,0);
@@ -2228,8 +2228,8 @@
 
 		int y = idx / 3;
 		int x = (idx % 3) * 26;
-		mvwprintw(dbg.win_var, y, x, dv->GetName());
-		mvwprintw(dbg.win_var, y,  (x + DEBUG_VAR_BUF_LEN + 1) , buffer);
+		mvwprintw(dbg.win_var, y, x, "%s", dv->GetName());
+		mvwprintw(dbg.win_var, y,  (x + DEBUG_VAR_BUF_LEN + 1) , "%s", buffer);
 	}
 
 	wrefresh(dbg.win_var);
--- a/src/debug/debug_gui.cpp
+++ b/src/debug/debug_gui.cpp
@@ -95,7 +95,7 @@
 		for (string::size_type posf=0, posl; (posl=(*i).find('\n',posf)) != string::npos ;posf=posl+1)
 			rem_lines -= (int) ((posl-posf) / maxx) + 1; // len=(posl+1)-posf-1
 		/* Const cast is needed for pdcurses which has no const char in mvwprintw (bug maybe) */
-		mvwprintw(dbg.win_out,rem_lines-1, 0, const_cast<char*>((*i).c_str()));
+		mvwprintw(dbg.win_out,rem_lines-1, 0, "%s", const_cast<char*>((*i).c_str()));
 	}
 	mvwprintw(dbg.win_out,maxy-1, 0, "");
 	wrefresh(dbg.win_out);
